crypto/bn254/keystore/keystore.go

```diff
@@ -0,0 +1,32 @@
+package keystore
+
+import (
+	"context"
+	"github.com/Layr-Labs/eigensdk-go/crypto/bls"
+	"github.com/Layr-Labs/eigensdk-go/types"
+)
+
+type Signer struct {
+	keyPair *bls.KeyPair
+}
+
+func New(path, password string) (*Signer, error) {
```

Comment:
NewFromFile and NewFromHex?

Reply:
this is always from File that's why keystore. maybe I can create another implementation for plaintext? but I don't want anyone to use plaintext 😅